### DISCUSSION TITLE
Need the ability to use PowerShell to publish Content Types from the Content Type Hub

### DISCUSSION THREAD

**Discussion by smchargue** [pnp/powershell#1522] [Ideas]
Would love to have the ability to publish a custom content type from the ContentTypeHub site to the rest of the tenant.  Right now our scripts create the site columns and content types, but we have to stop the script and go and publish the CT's manually - wgucg is time consuming and tedious.  If there ia a way to do it, I've haven't been able to find it (except for some ridiculous script floating around that gets an IntenetExplorere object and attempts to publish them); This isn't a one-off for us, we run these scripts on all our custom consulting projects. 

![image](https://user-images.githubusercontent.com/4064621/149403757-d7b38188-a4ae-48d8-8018-73201f0d0f8b.png)

**Comment by LeonArmston** [comment]
Hi

Doesn't https://github.com/pnp/powershell/blob/dev/documentation/Add-PnPContentTypesFromContentTypeHub.md do it?

Thanks

**Reply by smchargue** [reply]
I'll try anything, but looking at the command it suggest that the get *published* content types, and publishing them is the step I'm stuck on manually doing right now.
